collect_app/src/test/java/org/odk/collect/android/widgets/AnnotateWidgetTest.java

```diff
@@ -106,6 +107,23 @@ public void buttonsShouldLaunchCorrectIntents() {
         assertExtraEquals(DrawActivity.OPTION, DrawActivity.OPTION_ANNOTATE, intent);
     }
 
+    @Test
+    public void buttonsShouldLaunchCorrectIntentsWhenCustomPackageIsSet() {
```

Comment:
Instead of testing this in both the Annotate and Image widget classes, you could write one set for `ImageCaptureIntentCreator.imageCaptureIntent`.

Reply:
Yes but then I would need tests in both widgets to make sure that `ImageCaptureIntentCreator` is used. To do that I would need to pass it to those widgets (not use as a static method like now) to make it testable.
And also we have other tests that verify intents in those classes so I think the ones I added are similar and could be there.